evalml/automl/automl_search.py

```diff
@@ -424,8 +425,14 @@ def search(self, X, y, data_checks="auto", show_iteration_plot=True):
 
         if self.allowed_pipelines == []:
             raise ValueError("No allowed pipelines to search")
+
+        run_ensembling = self.ensembling
```

Comment:
I think we should pass `run_ensembling` to `IterativeAlgorithm` rather than `self.ensembling`?

Reply:
Hmmm, I think you make a really good point! I'll update this to pass in `run_ensembling` instead, but keep the changes in `IterativeAlgorithm` s.t. the two are not dependent on each other. Thanks for the great suggestion!